packages/vats/test/test-lib-board.js

```diff
@@ -49,6 +49,14 @@ test('makeBoard', async t => {
   t.is(idObj2b, 'tooboard012');
 });
 
+test('board values must be scalar keys', async t => {
```

Comment:
new behavior or just new test?

Reply:
just new test, I'm pretty sure. Since the map goes both ways, the values were always put in the key position of a collection. It's possible that the collection we used before VDO stuff didn't check the passtyle of the keys, but I'm pretty sure I've gotten "you can't stick that on the board" exceptions when I tried to use a record / object as a value.